### ISSUE TITLE
tor browser already has an option to change IP

### ISSUE THREAD

**Issue by mekb-turtle** [FDX100/Auto_Tor_IP_changer#12]
why do these low effort repos of yours get so many stars? also why are you assuming people use debian/apt, let them install the dependencies manually, they're not going to use this if they don't have tor

**Comment by Ali-Flt** [comment]
this can be installed in a server.

**Comment by mekb-turtle** [comment]
if someone is smart enough to set up a tor node on a dedicated server, i don't think they'd be using repos like these

**Comment by Keskebeu** [comment]
Yep this is a simple Python script. Read the script and you know that the code is a joke.

Don't use Tor proxies to hide your identity you are just simply sending your sensitive traffic to someone who has no life.

**Comment by mekb-turtle** [comment]
they even have the audacity to advertise their facebook :sob: also idk what `service` command is but if it's an init system, it would only work on that one lmfao

**Comment by Ali-Flt** [comment]
@mekb-turtle `service` command also works on systemd-based systems so no need to lyfao. 

I don't get why yall mad about this script. It works, its simple and it saves 10 minutes of your time that you would've spent writing it so I don't see what's wrong here.
